Resources/doc/reference/advanced_configuration.rst

```diff
@@ -120,6 +120,35 @@ application's config file:
                     model_manager:          # dependency name, from the table above
                         sonata.order.admin.order.manager  # customised service id
 
+
+Using Annotations to define Admins
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
https://github.com/sonata-project/SonataAdminBundle/pull/3624 << I added the change here